doc/ref_model/chapters/chapter07.rst

```diff
@@ -708,33 +709,91 @@ Access to the repository must be granted by a dedicated authorization. The code
 identified as described previously. After validating the software, it can be moved to the appropriate production 
 repository.
 
-SBOM
-~~~~
-
-It is crucial to identify the software components and their origins. The Software Bill of Materials (SBOM), 
-described by `US NTIA <https://www.ntia.gov/SBOM>`__ (National Telecommunications and Information Administration), 
-is an efficient tool to identify software components. The SBOM is an inventory of software components and the relationships 
-between them. NTIA describes how to establish an SBOM and provides SBOM standard data formats. In case of vulnerability 
-detected for a component, the SBOM inventory is an effective means to identify the impacted component and provide remediation.
-
-A transparent software supply chain offers benefits for vulnerabilities remediation, but also for licensing management and it 
-provides assurance of the source and integrity of components. In order to use the SBOM efficiently and spread its adoption, 
-information must be generated and shared in a standard format. This format must be machine-readable to allow automation. The 
-NTIA white paper `“Survey of existing SBOM formats and standards” <https://www.ntia.doc.gov/files/ntia/publications
-/ntia_sbom_formats_and_standards_whitepaper_-_version_20191025.pdf>`__ identifies two key formats covering baseline SBOM 
-information: Software Package Data eXchange (SDPX) and Software Identification Tags (SWID tags).
-
-`SPDX <https://spdx.dev/>`__ is an open-source machine-readable format developed under the umbrella of the Linux Foundation. 
-The `SPDX specification 2.2 <https://spdx.dev/specifications/>`__ has been published as the standard ISO/IEC 5962:2021. It 
-provides a language for communicating the data, licenses, copyrights, and security information associated with software 
-components. With the SPDX specification 2.2, multiple file formats are available: YAML, JSON, RDF/XML, tag\:value flat text, 
-and xlss spreadsheets.
-
-`SWID Tags <https://nvd.nist.gov/products/swid>`__ is an international XML-based standard used by commercial software 
-publishers and has been published as the standard ISO/IEC 19770-2. The specification defines four types of SWID tags: primary, 
-patch, corpus, and supplemental to describe a software component.
+Software Bill of Materials (SBOM)
```

Comment:
Do we want to add pointers at the end of this section to other open source forums in which best practices for establishing, adopting and implementing SBOMs are being driven? I am thinking of OpenSSF and OpenChain here. The Anuket RAs and RIs will likely refer to SBOMs (RA) or use them (RI), but the best practices are defined elsewhere.

Reply:
@gkunz Yes, we can add pointers to other open source forums. We will have to enrich this section over time. I propose to keep it for Nile release.

Reply:
Fine with me.